Makefile

```diff
@@ -524,8 +524,8 @@ api/openapi-spec/swagger.json: $(GOPATH)/bin/swagger dist/kubeified.swagger.json
 	swagger validate api/openapi-spec/swagger.json
 	go test ./api/openapi-spec
 
-api/jsonschema/schema.json: api/openapi-spec/swagger.json hack/jsonschema/main.go
-	go run ./hack/jsonschema
+api/jsonschema/schema.json: api/openapi-spec/swagger.json hack/jsonschema/main.py
+	./hack/jsonschema/main.py
```

Comment:
Could I please ask you do this in Go? I don't want to add another language to our stack that needs support.

Reply:
done